jwst/transforms/jwst_types.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+
+"""
+Defines a ``JWSTTransformType`` used by jwst pipeine transform types.
+All types are added automatically to ``_jwst_types`` and the JWSTExtension.
+
+"""
+import six
+
+from asdf.asdftypes import ExtensionTypeMeta
+from astropy.io.misc.asdf.tags.transform.basic import TransformType
+from astropy.io.misc.asdf.types import AstropyTypeMeta
+
+__all__ = ['JWSTTransformType']
+
+
+_jwst_types = set()
+
+
+class JWSTTypeMeta(AstropyTypeMeta):
```

Comment:
I think this should inherit `ExtensionTypeMeta`, not `AstropyTypeMeta`

Reply:
I get an error if it inherits from `ExtensionTypeMeta`.
```
E   TypeError: metaclass conflict: the metaclass of a derived class must be a (non-strict) subclass of the metaclasses of all its bases
```

Reply:
Basically because I need to inherit from `TransformType`.

Reply:
That's a little bit unfortunate, but yes you're right.